e2e/shielding.test.js

```diff
@@ -0,0 +1,38 @@
+const { log, device, by, expect } = require('detox');
+
+import { loadRecipientWallet } from './e2e-utils/loadRecipientWallet.js';
+
+const sleep = ms => new Promise(r => setTimeout(r, ms));
+
+describe('Renders wallet data correctly.', () => {
+  // i just pulled this seed out of thin air
+  it('loads a wallet', async () => await loadRecipientWallet());
+  it('shields transparent funds successfully', async () => {
+    // Shield Assets
+    await waitFor(element(by.id('header.shield')))
+      .toExist()
+      .withTimeout(50000);
+    await element(by.id('header.shield')).tap();
+    // // Confirm dialog
+    await expect(element(by.text('CONFIRM'))).toBeVisible();
+    await element(by.text('CONFIRM')).tap();
+    // Wait for confirmation
+    await waitFor(element(by.text(/Transmitted/gi)))
```

Comment:
IMO this test can fail here, because the `transmitted` stage can be really fast and sometimes you cannot see it in the UI. Here would be safer if you put `Transmitted` OR `In Mempool`.